recipes/mysql-connector-c/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mysql-connector-c" %}
+{% set version = "6.1.6" %}
+{% set sha256 = "2222433012c415871958b61bc4f3683e1ebe77e3389f698b267058c12533ea78" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://ftp.iij.ad.jp/pub/db/mysql/Downloads/Connector-C/{{ name|lower }}-{{ version }}-src.tar.gz
```

Comment:
What is this ftp server? I see it's the same url as in the bioconda reciple, but is it owned by MySQL? If not, what is its origin? How is it related to the source tarballs available at:

https://dev.mysql.com/downloads/connector/c/